app/features/analysis/cards/OneRM/tags/edit_tags/EditTagsModal.js

```diff
@@ -0,0 +1,383 @@
+// TODO: consider splitting this component into two different ones rather than using if statements everywhere
+
+import React, {Component} from 'react';
+import {
+    View,
+    Text,
+    TextInput,
+    TouchableHighlight,
+    TouchableOpacity,
+    Modal,
+    StyleSheet,
+    FlatList,
+    Platform
```

Comment:
- [ ] In general, I want to institute adding commas to the last one so there's fewer merge conflicts moving forward